toolkit/tools/scheduler/schedulerutils/printresults.go

```diff
@@ -188,6 +188,10 @@ func PrintBuildSummary(pkgGraph *pkggraph.PkgGraph, graphMutex *sync.RWMutex, bu
 	logger.Log.Infof("Number of failed SRPMs:            %d", len(failures))
 	logger.Log.Infof("Number of blocked SRPMs:           %d", len(unbuiltSRPMs))
 	logger.Log.Infof("Number of unresolved dependencies: %d", len(unresolvedDependencies))
+
+	logger.Log.Infof("Number of RPMs conflicts: : %d", len(rpmConflicts))
```

Comment:
Thanks for adding that line! One ask: we already print the conflicting toolchain packages (if there's anything to print) below in the `RPM Conflicts with toolchain` section. To be consistent with the summary's style and to avoid confusion, let's be strict about this message:

```suggestion
	logger.Log.Infof("RPM conflicts with toolchain: %d", len(rpmConflicts))
```